src/main/java/com/project/momskitchen/backend/services/Services.java

```diff
@@ -0,0 +1,9 @@
+package com.project.momskitchen.backend.services;
+
+import java.util.ArrayList;
+
+public interface Services {
```

Comment:
Unless you are planning to provide any sort of common functionality I don't think it is a good idea to generalize the contract of every service with the `Services` interface